org.knime.sdk.setup/KNIME-AP-nightly.target

```diff
@@ -1,24 +1,65 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde version="3.8"?>
-<target includeMode="feature" name="KNIME Analytics Platform (nightly)">
+<target includeMode="feature" name="KNIME Analytics Platform (nightly)" sequenceNumber="2023071200">
```

Comment:
What's the sequenceNumber for?

Reply:
KNIME was using this in their KNIME SDK setup, I think it is an optional thing to make something distinct from another file, e.g. when there is a change you can change the serial number. I used it now with the change date.